src/praisonai-agents/praisonaiagents/agent/chat_mixin.py

```
@@ -1042,25 +1042,17 @@ def _resolve_skill_invocation(self, prompt):
-                    try:
-                        registry.set_backend(
-                            AutoApproveBackend(),
-                            agent_name=agent_name,
-                            tool_name=_tn,
-                        )
-                    except TypeError:
-                        # Older registry may not accept tool_name kwarg
-                        registry.set_backend(AutoApproveBackend(), agent_name=agent_name)
+                    registry.auto_approve_tool(_tn, agent_name=agent_name)
         except Exception:  # pragma: no cover - approval is optional
             pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)